src/me/coley/recaf/ui/component/editor/InsnListEditor.java

```diff
@@ -198,6 +193,55 @@ private void setupModel() {
 					getItems().removeAll(selectionModelProperty().getValue().getSelectedItems());
 				}
 			});
+			addEventHandler(KeyEvent.KEY_RELEASED,e -> {
+				ConfKeybinds keys = ConfKeybinds.instance();
+				if (keys.active && !e.isControlDown()) {
+					return;
+				}
+				if (e.getCode() != KeyCode.F) {
+					return;
+				}
+				Dialog<String> contentSearchDialog = new Dialog<>();
```

Comment:
For window consistency _(Icon/style/global-keybinds)_ Using `JavaFx.stage(scene, title, top=true)` to create the window would be good.

OR

Creating an alternative `JavaFx.stage(scene, title, top=true)` but for creating `Dialog`s would be good too.

***

I know this would require changing the logic of `dialog.showAndWait()` to get the text result, but that's not too big of a change in my opinion.